scripts/reindex.py

```diff
@@ -105,12 +106,19 @@ def main(argv: List[str]):
 
     if args.index:
         logger.info('Queuing notifications for reindexing ...')
+        num_notifications = 0
         for catalog in args.catalogs:
             if args.partition_prefix_length:
                 azul.remote_reindex(catalog, args.prefix, args.partition_prefix_length)
+                num_notifications = None
             else:
-                azul.reindex(catalog, args.prefix)
+                num_notifications += azul.reindex(catalog, args.prefix)
         if args.wait:
+            if num_notifications == 0:
+                logger.error('Notifications not matched for prefix %r and %r catalogs',
+                             args.catalogs,
+                             args.prefix)
+                raise AzulClientNotificationError
```

Comment:
Print warning level message "No notifications  for prefix %r were send out for catalogs %r, nothing to do!"